bot.py

```diff
@@ -1,107 +1,120 @@
 import asyncio
 import logging
 import sys
-import traceback
+import typing
+from typing import Annotated, Optional, Literal
 
 import discord
+from discord.ext import commands
 from dynaconf import ValidationError
 
+import constants
 import lib
-from messages import SubmitMessage, GetBestTimesMessage
-from database import Database
 from config import settings
-import constants
+from database import Database
+from error_handler import ErrorHandlerCog
 
 logger = logging.getLogger(__name__)
 
 
-class MyBot(discord.Client):
+class MyBot(commands.Bot):
     def __init__(self, *args, **kwargs):
```

Comment:
we should probably add `__slots__`, its a perf benefit but it also helps typecheckers assert what can be a member of the class
```suggestion
    __slots__ = ("queue", "db")
    
    def __init__(self, *args, **kwargs):
```